ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/deprovision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-deprovision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-ovn-hypershift-deprovision
+  steps:
+  - chain: hypershift-azure-destroy
+  - chain: cucushift-installer-rehearse-azure-ipi-deprovision
+  env:
+    - name: HYPERSHIFT_BASE_DOMAIN
+      default: "qe.azure.devcluster.openshift.com"
+    - name: HYPERSHIFT_AWS_REGION
```

Comment:
aws region ?  change to azure....

Reply:
https://github.com/openshift/release/pull/33460 wait for merge